forum/models/abstract.py

```diff
@@ -10,37 +10,49 @@
 class AbstractModel(ABC):
     """Abstract Class for Mongo model implementation"""
 
-    def __init__(self, collection_name):
-        db_name = 'cs_comments_service'
+    def __init__(self, collection_name, client=None):
         self.client = MongoBackend(collection=collection_name)
```

Comment:
Shouldn't this be:

    self.client = client or MongoBackend(collection=collection_name)
Comment

Reply:
Updated